CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 - Deprecated `channel.follow` v1 eventsub event
 - Deprecated `Get User Follows` and associated follower related extension methods
 - Deprecated Twitch-defined tags: `Get All Stream Tags`, `Get Stream Tags`, `Replace Stream Tags` and `TwitchTag`
+- Updated `TWITCH_EVENTSUB_WEBSOCKET_URL` constant
```

Comment:
this is not needed, as it's still in beta, was introduced in 0.7.0 and hasn't been added to the changelog yet